website/pages/docs/security/security-models/core.mdx

```diff
@@ -0,0 +1,380 @@
+
+- **Developer** - This is someone who is responsible for creating, and possibly deploying applications connected, or 
+  configured with Consul. In some cases they may have no access, or limited capabilities to view Consul information, 
+  such as through metrics, or logs.
+
+- **User** - This is the end user, using applications backed by services managed by Consul. In some cases services may 
+  be public facing on the internet such as a web server, typically through a load-balancer, or ingress gateway. This is 
+  someone who should not have any network access to the Consul agent APIs.
+
+### Secure Configuration
+
+Consul's security model is applicable only if all parts of the system are running with a secure configuration; **Consul 
+is not secure-by-default.** Without the following mechanisms enabled in Consul's configuration, it may be possible to 
+abuse access to a cluster. Like all security considerations, administrators must determine what is appropriate for their
+environment and adapt these configurations accordingly.
+
+#### Requirements
+
+- **mTLS** - Mutual authentication of both the TLS server and client x509 certificates prevents internal abuse through 
+  unauthorized access to Consul agents within the cluster.
+
+  - [`verify_incoming`](/docs/agent/options#verify_incoming) - By default this is false, and should almost always be set
+    to true to require TLS verification for incoming client connections. This applies to both server RPC and to the 
+    HTTPS API.
+
+  - [`verify_incoming_https`](/docs/agent/options#verify_incoming_https) - By default this is false, and should be set 
+    to true to require clients to provide a valid TLS certificate when the Consul HTTPS API is enabled.
+
+  - [`verifing_incoming_rpc`](/docs/agent/options#verify_incoming_rpc) -  By default this is false, and should almost 
+    always be set to true to require clients to provide a valid TLS certificate for Consul agent RPCs.
+
+  - [`verify_outgoing` - By default this is false, and should be set to true to require TLS for outgoing connections from
+    server or client agents. Servers that specify `verify_outgoing = true` will always talk to other servers over TLS, 
+    but they still accept non-TLS connections to allow for a transition of all clients to TLS. Currently the only way to 
+    enforce that no client can communicate with a server unencrypted is to also enable `verify_incoming` which requires 
+    client certificates too.
+
+  - `enable_agent_tls_for_checks`](/docs/agent/options#verify_outgoing) - By default this is false, and should almost 
+    always be set to true to require mTLS to set up the client for HTTP or gRPC health checks. This was added in 
+    Consul 1.0.1.
+
+  - `verify_server_hostname` - By default this is false, and should be set to true to require for outgoing TLS 
+    connections that the TLS certificate presented by the servers matches `server.<datacenter>.<domain> hostname`. The 
```

Comment:
This blob reads better to me if two chunks are swapped

```
By default this is false, and should be set to true to require for outgoing TLS
connections that the TLS certificate presented by the servers matches
`server.<datacenter>.<domain> hostname`.
```
becomes
```
By default this is false, and should be set to true to require
that the TLS certificate presented by the servers matches
`server.<datacenter>.<domain> hostname` for outgoing TLS connections.
```

Reply:
Reads better to me too, thanks!